### PR TITLE
Add crimson log variants

### PR DESCRIPTION
**Pull request by haykam821**
This adds the following variants of the crimson log:

* Stripped crimson log
* Crimson wood
* Stripped crimson wood

In addition, like vanilla, all of these can be crafted into planks.

### CODE REVIEW

File: src/main/java/modfest/teamgreen/content/block/CrimsonStrippedBlocks.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package modfest.teamgreen.content.block;
+
+import com.google.common.collect.ImmutableMap;
+
+import modfest.teamgreen.mixin.AccessorAxeItem;
+import net.minecraft.block.Block;
+
+public class CrimsonStrippedBlocks {
+	public static void init() {
+		ImmutableMap.Builder<Block, Block> builder = ImmutableMap.builder();
```
Comment (valoeghese) [inline src/main/java/modfest/teamgreen/content/block/CrimsonStrippedBlocks.java:10]: quality code 10/10

File: src/main/java/modfest/teamgreen/mixin/AccessorAxeItem.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Mutable;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+import net.minecraft.block.Block;
+import net.minecraft.item.AxeItem;
+
+@Mixin(AxeItem.class)
+public interface AccessorAxeItem {
```
Comment (valoeghese) [inline src/main/java/modfest/teamgreen/mixin/AccessorAxeItem.java:13]: fabric api should have this

File: src/main/resources/assets/crimson/blockstates/stripped_crimson_log.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+	"variants": {
```
Comment (valoeghese) [inline src/main/resources/assets/crimson/blockstates/stripped_crimson_log.json]: 2 space indents for json

File: src/main/resources/assets/crimson/blockstates/stripped_crimson_wood.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+	"variants": {
```
Comment (valoeghese) [inline src/main/resources/assets/crimson/blockstates/stripped_crimson_wood.json]: ditto

File: src/main/resources/assets/crimson/lang/en_us.json [hunk 5]
```diff
@@ -22,6 +22,9 @@
   "block.crimson.vanadinite_ore": "Vanadinite Ore",
   "block.crimson.crimson_thorn": "Crimson Thorn",
   "block.crimson.crimson_log": "Crimson Log",
+	"block.crimson.stripped_crimson_log": "Stripped Crimson Log",
+  "block.crimson.crimson_wood": "Crimson Wood",
+	"block.crimson.stripped_crimson_wood": "Stripped Crimson Wood",
```
Comment (valoeghese) [inline src/main/resources/assets/crimson/lang/en_us.json]: wtf is this indentation

File: src/main/resources/crimson.mixins.json [hunk 6]
```diff
@@ -3,6 +3,7 @@
   "package": "modfest.teamgreen.mixin",
   "compatibilityLevel": "JAVA_8",
   "mixins": [
+		"AccessorAxeItem",
```
Comment (valoeghese) [inline src/main/resources/crimson.mixins.json]: wtf

File: src/main/resources/data/crimson/recipes/crafting_crimson_planks.json [hunk 7]
```diff
@@ -2,7 +2,7 @@
     "type": "minecraft:crafting_shapeless",
     "ingredients": [
         {
-            "item": "crimson:crimson_log"
+            "tag": "crimson:crimson_logs"
```
Comment (valoeghese) [inline src/main/resources/data/crimson/recipes/crafting_crimson_planks.json:5]: may as well change the indentation to 2 spaces while you're here

File: src/main/resources/data/crimson/recipes/crafting_crimson_wood.json [hunk 8]
```diff
@@ -0,0 +1,17 @@
+{
+	"type": "minecraft:crafting_shaped",
+	"group": "bark",
+	"pattern": [
```
Comment (valoeghese) [inline src/main/resources/data/crimson/recipes/crafting_crimson_wood.json]: yuck tabs in json

File: src/main/resources/data/crimson/tags/blocks/logs.json [hunk 9]
```diff
@@ -1,5 +1,5 @@
 {
- "values": [
-    "crimson:crimson_log"
- ]
+	"values": [
+		"#crimson:crimson_logs"
```
Comment (valoeghese) [inline src/main/resources/data/crimson/tags/blocks/logs.json]: 2 spaces in json

File: src/main/resources/data/crimson/tags/items/logs.json [hunk 10]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (valoeghese) [inline src/main/resources/data/crimson/tags/items/logs.json:1]: 2 spaces in json

File: src/main/resources/data/minecraft/tags/blocks/logs.json [hunk 11]
```diff
@@ -1,6 +1,6 @@
 {
- "replace": false,
- "values": [
-    "#crimson:logs"
- ]
+	"replace": false,
```
Comment (valoeghese) [inline src/main/resources/data/minecraft/tags/blocks/logs.json]: "

Comment (valoeghese) [reply src/main/resources/data/minecraft/tags/blocks/logs.json]: (ditto mark)

**Review by valoeghese — CHANGES_REQUESTED** [review]

File: src/main/resources/data/crimson/recipes/crafting_crimson_planks.json (hunk 7 above)
Comment (haykam821) [reply src/main/resources/data/crimson/recipes/crafting_crimson_planks.json:5]: This file already uses 2 spaces for indentation.

File: src/main/resources/assets/crimson/blockstates/stripped_crimson_log.json (hunk 3 above)
Comment (haykam821) [reply src/main/resources/assets/crimson/blockstates/stripped_crimson_log.json]: Fixed in 500f65b0a3f45983d78cbaa9d5223dbddf1fea9b.

File: src/main/resources/assets/crimson/blockstates/stripped_crimson_wood.json (hunk 4 above)
Comment (haykam821) [reply src/main/resources/assets/crimson/blockstates/stripped_crimson_wood.json]: Fixed in 500f65b0a3f45983d78cbaa9d5223dbddf1fea9b.

File: src/main/resources/assets/crimson/lang/en_us.json (hunk 5 above)
Comment (haykam821) [reply src/main/resources/assets/crimson/lang/en_us.json]: Fixed in 500f65b0a3f45983d78cbaa9d5223dbddf1fea9b.

File: src/main/resources/crimson.mixins.json (hunk 6 above)
Comment (haykam821) [reply src/main/resources/crimson.mixins.json]: Fixed in 500f65b0a3f45983d78cbaa9d5223dbddf1fea9b.

File: src/main/resources/data/crimson/recipes/crafting_crimson_wood.json (hunk 8 above)
Comment (haykam821) [reply src/main/resources/data/crimson/recipes/crafting_crimson_wood.json]: Fixed in 500f65b0a3f45983d78cbaa9d5223dbddf1fea9b.

File: src/main/resources/data/crimson/tags/blocks/logs.json (hunk 9 above)
Comment (haykam821) [reply src/main/resources/data/crimson/tags/blocks/logs.json]: Fixed in 500f65b0a3f45983d78cbaa9d5223dbddf1fea9b.

File: src/main/resources/data/crimson/tags/items/logs.json (hunk 10 above)
Comment (haykam821) [reply src/main/resources/data/crimson/tags/items/logs.json:1]: Fixed in 500f65b0a3f45983d78cbaa9d5223dbddf1fea9b.

File: src/main/resources/data/minecraft/tags/blocks/logs.json (hunk 11 above)
Comment (haykam821) [reply src/main/resources/data/minecraft/tags/blocks/logs.json]: Fixed in 500f65b0a3f45983d78cbaa9d5223dbddf1fea9b.

File: src/main/java/modfest/teamgreen/mixin/AccessorAxeItem.java (hunk 2 above)
Comment (haykam821) [reply src/main/java/modfest/teamgreen/mixin/AccessorAxeItem.java:13]: FabricMC/fabric#576 exists, but it hasn't been merged yet.

File: src/main/resources/data/crimson/recipes/crafting_crimson_wood.json (hunk 8 above)
Comment (valoeghese) [reply src/main/resources/data/crimson/recipes/crafting_crimson_wood.json]: for some reason it seems to be not showing up idk but I can see it exists in the actual repo

**Review by valoeghese — APPROVED** [review]
Honestly bikeshed is bad so I'm gonna merge this

**Pull request merged**